From 1cbc52e090d54b9014ce3f5517f6c12bb955fa9f Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 20:42:47 -0500
Subject: [PATCH] Install the package.xml manifest

---
 rpm/ros-rolling-fastcdr.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ros-rolling-fastcdr.spec b/rpm/ros-rolling-fastcdr.spec
index c900f5898cad..8e9e8d6d763b 100644
--- a/rpm/ros-rolling-fastcdr.spec
+++ b/rpm/ros-rolling-fastcdr.spec
@@ -65,6 +65,7 @@ CDR serialization implementation.
 
 %install
 %cmake_install
+install -m0644 -p -D package.xml %{buildroot}%{bloom_prefix}/share/%{pkg_name}/package.xml
 
 
 %if 0%{?with_tests}
-- 
2.44.0

